Tryouts/Plugins/Services/CommunicationsServices/DwmUtils/DwmVisual.cs

```diff
@@ -0,0 +1,273 @@
+﻿using Dwm;
+using MorganStanley.ComposeUI.Tryouts.Core.Abstractions;
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Input;
+using System.Windows.Interop;
+using System.Windows.Media;
+using Subscriptions;
+using MorganStanley.ComposeUI.Tryouts.Visuals.WindowsOnly.Win32Calls;
+
+namespace MorganStanley.ComposeUI.Services.CommunicationsServices.DwmUtils
```

Comment:
What does DWM have to do with Communications Services?

Reply:
I understand that the DWM implementation would rely on communication but here it seems DwmUtils were added directly into the communication service. Shouldn't the two be separated?